logic/src/commonMain/kotlin/com/wire/kalium/logic/data/message/ProtoContentMapper.kt

```diff
@@ -361,6 +358,32 @@ class ProtoContentMapperImpl(
         )
     }
 
+
+    private fun unpackEphemeral(
+        protoContent: GenericMessage.Content.Ephemeral
+    ): MessageContent.FromProto {
+        val messageContent = when (val ephemeralContent = protoContent.value.content) {
+            is Ephemeral.Content.Text -> {
+                val genericMessageTextContent = GenericMessage.Content.Text(
+                    Text(
+                        content = ephemeralContent.value.content,
+                        mentions = ephemeralContent.value.mentions,
+                        quote = ephemeralContent.value.quote,
+                        expectsReadConfirmation = ephemeralContent.value.expectsReadConfirmation
+                    )
+                )
+                unpackText(genericMessageTextContent)
+            }
+
+            // TODO: Mateusz : for now we ignore anything else then Text
```

Comment:
From an unpacking point of view, we could handle at least `Asset` for now, no?

Reply:
I plan to handle it next, after covering it with Unit tests